tests/unit_tests/test_event_service.py

```diff
@@ -37,6 +39,22 @@ def service(api_accessor: ApiAccessor):
     ]
 
     result = await service.execute_batch_update(42, queue)
-    assert result == json.loads(content.decode('utf-8'))['updated_events']
 
-    service.api_accessor.api_post.assert_called_once_with("/events/recordMultiple", 42, data=dict(updates=queue))
+    events_data = []
+    for event in json.loads(content)['data']:
+        converted_event = dict(
+            event_id=event['attributes']['eventId'],
+            count=event['attributes']['count']
+        )
+        events_data.append(converted_event)
+
+
+    assert result == events_data
+
+    service.api_accessor.update_events.assert_called_once_with(queue, 42)
```

Comment:
`(queue, 42)` - space

Reply:
what ? there should not be space ? or there should be space ?

Reply:
space betwen comma and number

Reply:
It's already there 
![image](https://user-images.githubusercontent.com/33836535/49408861-7cf1ad80-f75e-11e8-8993-010a9dae301d.png)